src/js/models/section.js

```diff
@@ -0,0 +1,69 @@
+export const DEFAULT_TAG_NAME = 'p';
+export const SECTION_TAG_NAMES = [
+  'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'p', 'div'
```

Comment:
Seems like this should be the same list as https://github.com/bustlelabs/content-kit-editor/blob/summertime/src/js/parsers/dom.js#L7